frontend/packages/ceph-storage-plugin/src/odf-external-providers/external-providers.ts

```diff
@@ -0,0 +1,15 @@
+import { ExternalProvider } from './types';
+import { RHCS, StorageClusterIdentifier } from '../constants/create-storage-system';
+
+export const ODF_EXTERNAL_PROVIDERS: ExternalProvider[] = [
```

Comment:
@vbnrh we don't want to do that.
This file is to allow external providers to add their components e.g flash system.
I am keeping it separate so that I can add more details to it and maintain separate components for the external system.

Reply:
Yeah we want to add support for each of providers to have their own folder and add their dedicated step.